scripts/lldpshow

```diff
@@ -80,9 +80,11 @@ class Lldpshow(object):
             lldp_interface_list = lldp_port if lldp_port is not None else self.lldp_interface[lldp_instace_num]
             # In detail mode we will pass interface list (only front ports) and get O/P as plain text
             # and in table format we will get xml output
-            lldp_cmd = 'sudo docker exec -i lldp{} lldpctl '.format(self.lldp_instance[lldp_instace_num]) + (
-                '-f xml' if not lldp_detail_info else lldp_interface_list)
-            p = subprocess.Popen(lldp_cmd, stdout=subprocess.PIPE, shell=True, text=True)
+            lldp_args = ['-f', 'xml'] if not lldp_detail_info else [lldp_interface_list]
```

Comment:
>lldp_interface_list [](http://example.com/codeflow?start=68&length=19)

you can check `lldp_interface_list == ''` and assign different values to lldp_args.

So this line will be come a if-elsif-else block.